package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "cache-manager",
```

```suggestion
  "name": "@janiscommerce/cache-manager",
```
Agregar el dominio en el nombre del package